courses/free-courses-hi.md

```diff
@@ -147,7 +147,7 @@
 
 ### <a id="cpp"></a>C++
 
-* [C++ and DSA Foundation Course](https://youtube.com/playlist?list=PLxgZQoSe9cg0df_GxVjz3DD_Gck5tMXAd&si=frggLn7z_DWtYv6I) - College Wallah
+* [C++ and DSA Foundation Course](https://youtube.com/playlist?list=PLxgZQoSe9cg0df_GxVjz3DD_Gck5tMXAd&si=7xqizlHygwU8bMhr) - College Wallah
```

Comment:
```suggestion
* [C++ and DSA Foundation Course](https://youtube.com/playlist?list=PLxgZQoSe9cg0df_GxVjz3DD_Gck5tMXAd) - College Wallah
```
when you've made changes, please add `(changes made)` to the title of your PR

Reply:
Sure, I'll make sure to add '(changes made)' to the title of my PR after making the necessary updates. Thank you for the guidance!